apps/experiments/views/experiment.py

```diff
@@ -116,12 +116,16 @@ def get_queryset(self):
 
 
 class ExperimentForm(forms.ModelForm):
+    PROMPT_HELP_TEXT = """
+        Use {source_material} to place source material in the prompt. Use {participant_data} to place participant
+        data in the prompt.
+    """
```

Comment:
nit: this could be put with the other help texts in `Meta.help_texts` (unless that doesn't work for some reason)

Reply:
oh right, that's a thing. Thanks

Reply:
Hmm, it does seem to not pick it up from Meta if we declare the field as an attribute